tests/test_integration_address.py

```diff
@@ -675,24 +674,209 @@ def test_create_unit_address(self):
         # NOW CREATE IT
 
         self.assertRequestResponse(
-            '/service/ou/{}/create'.format(unitid),
+            '/service/ou/{}/edit'.format(unitid),
```

Comment:
Is edit used to create addresses for org units?

Reply:
From my understanding, `create` is used to create the unit itself, whereas all details are modified using the `edit` endpoint. Still, this test doesn't actually manage to add any addresses — considering that editing doesn't work yet — so I'll remove it.

Reply:
As far as I can see, we use the `create` endpoint for creating addresses for employees, so I think we should try to along the workflow for `e` and `ou` as much as we can.